src/toolbox/useTranslation.js

```diff
@@ -23,6 +28,12 @@ const LOCALE_EN = {
     all_words: ", ",
     two_words: " and ",
     last_word: ", and "
+  },
+  title: {
+    of_selected_cut_members: "of Selected {members} Members",
+    top_drilldowns: "for Top {drilldowns}",
+    by_drilldowns: "by {drilldowns}",
+    over_time: "Over Time"
   }
```

Comment:
i18next has a standard of using double brackets for variables: https://www.i18next.com/translation-function/interpolation

Reply:
isn't this something that would have to change in the @datawheel/use-translation package? Is that something we want to actually change?

Reply:
Confirmed -- change needed in dependency pkg. Will wait to change until that pkg is updated.